tests/autoyast/installation.pm

```diff
@@ -133,6 +133,8 @@ sub run {
       = qw(bios-boot nonexisting-package reboot-after-installation linuxrc-install-fail scc-invalid-url warning-pop-up inst-betawarning autoyast-boot);
     push @needles, 'autoyast-confirm'        if get_var('AUTOYAST_CONFIRM');
     push @needles, 'autoyast-postpartscript' if get_var('USRSCR_DIALOG');
+    # grub2 needle check is unreliable (stalls during timeout) - poo#28648
+    push @needles, 'linux-login-casp' if is_caasp;
```

Comment:
@rwx788 can you check?

Reply:
Hmm, I'm not sure why we need it here at all as we return after first stage and this tag is not explicitly used anywhere. So I guess this change should not be here. It's not related to the problem at least and we can create needle with reboot-after-installation.

Reply:
@rwx788 - we don't have reboot after installation 10 timeout on CaaSP, system just reboots. If there is stall then autoyast/installation.pm does not notice installation is finished (bios-boot needle disappears during stall)

Reply:
I still do not fully understand how it helps. Is it enough just to put extra needle into the array? I mean we don't use this tag anywhere later in the module. And regarding reboot-after-installation we have 3 tags which work as a condition to exit installer loop and return in case of caasp. Could you please explain how this particular change is related?

Reply:
I finally understood your point - thanks :) I added `linux-login-casp` to exit conditions, is it ok like this?